src/GOODFFunctions.h

```diff
@@ -26,6 +26,11 @@
 #include <wx/filename.h>
 #include <vector>
 #include "GOODF.h"
+// macro to both show and raise the LogWindow
```

Comment:
Revert this macro and all places where it's called as it currently doesn't do anything better than the existing triggering line.